<?xml version="1.0"?>
<launch>
    <node name ="xian_dj_tele_op_params_server" pkg="xian_dj_remote_operation_param_pkg" type="xian_dj_tele_op_params_server" />
    <node name ="joy_node" pkg="joy" type="joy_node" />
    <node name ="xian_dj_tele_op_controller_client" pkg="xian_dj_remote_operation_control_pkg" type="xian_dj_tele_op_controller_client" />
</launch>